docs/_functions/global/getConfiguredDomains.md

```diff
@@ -0,0 +1,62 @@
+---
+name: getConfiguredDomains
+parameters:
+  - name
+  - modifiers...
+---
+
+`getConfiguredDomains` is a simple helper function to return back configured domains until the point where the function
```

Comment:
Suggested description:

`getConfiguredDomains` is a helper function that returns the domain names configured at the time the function is called.  Calling this function early or later in `dnsconfig.js` may return different results.  Typical usage is to iterate over all domains at the end of your configuration file.